pkgs/development/libraries/libAfterImage/default.nix

```diff
@@ -54,6 +54,13 @@ stdenv.mkDerivation {
       # workaround '-p0' patchflags below.
       stripLen = 1;
     })
+
+    # fix https://github.com/root-project/root/issues/10990
```

Comment:
```suggestion
    # https://github.com/root-project/root/pull/11243, fixes https://github.com/root-project/root/issues/10990
```

Reply:
let's let upstream track the development issues such as PRs